src/tests/ftest/util/fio_test_base.py

```diff
@@ -127,7 +127,7 @@ def _start_dfuse(self):
                            exc_info=error)
             self.fail("Unable to launch Dfuse.\n")
 
-    def execute_fio(self):
+    def execute_fio(self, directory=None, stop_dfuse=True):
```

Comment:
It would be great if you could add the parameter descriptions.

Reply:
Done